core/src/genesis.rs

```diff
@@ -342,9 +342,18 @@ pub mod config {
 
     impl Default for GenesisConfiguration {
         fn default() -> Self {
+            let public_key: PublicKey =
+                "ed01204cffd0ee429b1bdd36b3910ec570852b8bb63f18750341772fb46bc856c5caaf"
+                    .parse()
+                    .expect("Public key not in mulithash format");
+            let private_key = PrivateKey::from_hex(
+                iroha_crypto::Algorithm::Ed25519,
+                "d748e18ce60cb30dea3e73c9019b7af45a8d465e3d71bcc9a5ef99a008205e534cffd0ee429b1bdd36b3910ec570852b8bb63f18750341772fb46bc856c5caaf"
+            ).expect("Private key not hex encoded");
+
             Self {
-                account_public_key: None,
-                account_private_key: None,
+                account_public_key: Some(public_key),
+                account_private_key: Some(private_key),
```

Comment:
Maybe should be left as `None`? Any thoughts?

Reply:
I'd leave it `None` and raise an error rather than accept a non-personalized configuration in the future

Reply:
you'd leave `account_private_key` as `None` for `Default` configuration? As you can see elsewhere, I've added keys that we use in sample files to `Default` configuration

Reply:
I can leave both as `None`, compare it to `sumeragi::Configuration::Default` for example

Reply:
Depending on what we do, it may be preferable to `impl Default for PublicKey`. 

So my reasoning is as follows. Having a non-customised configuration is a costly mistake in production. We only need the default implementation for the generator and for the tests. To use both, we could create a feature flag (`placeholder_defaults` sounds about right), and make un-customised configs unrepresentable without that feature flag. 

We would need to work around our method of reading environment variables for that to work, but I think it's worth the effort in the long run.

Reply:
here is an https://github.com/hyperledger/iroha/issues/2118 I just opened. We have a potential security issue on our hands with default configurations

Reply:
> Depending on what we do, it may be preferable to impl Default for PublicKey.

I don't think so, because there is a different public key for `client::Configuration` vs `cli::Configuration`. I would leave it without a default and define it in the implementation of the `Default` for the wrapping container as is done in this PR

Reply:
Have you considered transparent wrappers with unique `impl Default`? Adding the delineation at the type level as well as in documentation?

Reply:
I'll add you suggestion to the issue

Reply:
> So my reasoning is as follows. Having a non-customised configuration is a costly mistake in production. We only need the default implementation for the generator and for the tests. To use both, we could create a feature flag (placeholder_defaults sounds about right), and make un-customised configs unrepresentable without that feature flag.

I don't like introducing feature flag because it's part of the public api. Tests and generators can just call the configuration constructor or dump to a file that will get deserialized, or have a predefined structure which will implement `Into<Configuration>`, etc. There are many ways to do it without implementing `Default` for configurations